android/gradle.properties

```diff
@@ -18,8 +18,10 @@ org.gradle.jvmargs=-Xmx4096m -XX:+HeapDumpOnOutOfMemoryError
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 org.gradle.parallel=true
 
-CELO_RELEASE_STORE_FILE=celo-release-key.keystore
-CELO_RELEASE_KEY_ALIAS=celo-key-alias
+# Note that the key alias lacks Mobile Stack naming, since updating this would
+# require regenerating the keystore.
+MOBILESTACK_RELEASE_STORE_FILE=mobilestack-release-key.keystore
+MOBILESTACK_RELEASE_KEY_ALIAS=celo-key-alias
```

Comment:
I think we can update the keystore. Also an opportunity to use the same password for the store and the key inside.

Reply:
@jeanregisser What's the typical process for this? I assume we want to re-use our existing upload key; I can't seem to find an option in Android Studio (which is what I've used in the past for creating brand new upload keys/keystores) to create a new keystore from an existing key.

Reply:
Hey, this can be done using the `keytool` cli:
- Renaming the alias: https://stackoverflow.com/a/5707518/158525
- Changing the passwords: https://stackoverflow.com/a/56382758/158525